package.json

```diff
@@ -83,7 +83,8 @@
   "resolutions": {
     "strip-ansi": "^6.0.1",
     "string-width": "^4.2.2",
-    "wrap-ansi": "^7.0.0"
+    "wrap-ansi": "^7.0.0",
+    "axios": "^1.12.0"
```

Comment:
Is this because some dependencies are using a vulnerable version? Do we know which dependencies?

Reply:
it's -> wait-on 8.0.4

Reply:
Ah you're aleady on it!